dll/win32/syssetup/lang/ro-RO.rc

```diff
@@ -120,9 +121,9 @@ CAPTION "Asistent de instalare ReactOS"
 FONT 8, "MS Shell Dlg"
 BEGIN
     LTEXT "Componente în curs de înregistrare", IDC_TASKTEXT1, 65, 27, 232, 8
-    LTEXT "Installing start menu items", IDC_TASKTEXT2, 65, 39, 232, 8
-    LTEXT "Saving settings", IDC_TASKTEXT3, 65, 51, 232, 8
-    LTEXT "Removing temporary files", IDC_TASKTEXT4, 65, 63, 232, 8
+    LTEXT "Instalarea elementelor din meniul de pornire", IDC_TASKTEXT2, 65, 39, 232, 8
+    LTEXT "Salvarea setărilor", IDC_TASKTEXT3, 65, 51, 232, 8
```

Comment:
Please consider avoiding both „a salva” and „setări” (see 15th point in [Romanian translation notes.txt](https://github.com/reactos/reactos/blob/master/media/doc/Romanian%20translation%20notes.txt)) For translating „settings” maybe „opțiuni” also serves as a good alternative ehm... option.